docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -922,6 +924,58 @@ the following steps:
        jenkins:
            use-crumb: false
 
+Upgrading Jenkins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In order to upgrade Jenkins to a newer version, you need to rebuild the Docker image targeting the new version.
+The stable LTS versions can be viewed through the `changelog <https://www.jenkins.io/changelog-stable/>`__. and the corresponding
+Docker image can be found on `dockerhub <https://hub.docker.com/r/jenkins/jenkins/tags?page=1&ordering=last_updated>`__.
+
+1. Open the Jenkins Dockerfile and replace the value of ``FROM`` with ``jenkins/jenkins:lts``. This will use the latest LTS version.
```

Comment:
```suggestion
1. Open the Jenkins Dockerfile and replace the value of ``FROM`` with ``jenkins/jenkins:lts``. After running the command ``docker pull jenkins/jenkins:lts``, this will use the latest LTS version in the following steps.
```